doc/reference/orm.rst

```diff
@@ -1,25 +1,53 @@
 :banner: banners/orm_api.jpg
 
-.. _reference/orm:
-
 =======
 ORM API
```

Comment:
Suggestion for a more natural structure : 
- Models: base building block of a module ; depends on nothing
- Fields: logical continuation of models ; depends on models
- Recordsets: instantiation of models & fields ; depends on models, fields
- Method decorators: advanced use of fields ; depends on fields, recordsets
- Environment: depends on models, recordsets
- Common ORM methods: depends on models, recordsets, environment
- Search Domains: depends on ORM methods
- Inheritance and extension: advanced use of models ; depends on models, fields
- Error management: depends on nothing